lang/py3/setup.py

```diff
@@ -145,6 +142,11 @@ def Main():
       license = 'Apache License 2.0',
       keywords = 'avro serialization rpc',
       url = 'http://avro.apache.org/',
+      classifiers=[
+          'License :: OSI Approved :: Apache Software License',
+          'Programming Language :: Python :: 3 :: Only',
+      ],
```

Comment:
Can we be more explicit in the Python versions here? What version of Python 3.x are we going to support?

Reply:
I don't know the answer to this. Who makes such a decision.

Reply:
The community. We can propose a VOTE on the mailing list to enforce a minimal version Python 3: https://www.apache.org/foundation/voting.html#votes-on-code-modification

Reply:
I am OK with that, but get so little response from the dev ml that I am never actually sure if my posts land or not.

Reply:
A good idea is to add some kind of deadline. It is common with these vote processes to keep it open for 72 hours or so, after that, a decision is made.

Reply:
One way to determine this is to look at the download data from pypi and see which versions of python people are using when they install this library. The results for the past 30 days are the following:

```
[
  {
    "python_version": "3.5",
    "download_count": "30258"
  },
  {
    "python_version": "3.6",
    "download_count": "26228"
  },
  {
    "python_version": "3.4",
    "download_count": "9140"
  },
  {
    "python_version": "3.7",
    "download_count": "2502"
  },
  {
    "python_version": null,
    "download_count": "793"
  },
  {
    "python_version": "2.7",
    "download_count": "173"
  },
  {
    "python_version": "2.6",
    "download_count": "3"
  }
]
```

So it seems it is used commonly on 3.5+ and there is some use on 3.4.

Reply:
I have posted to the dev list. It has not appeared on the archives yet. Please let me know if you saw my post.